cmd/addvalidators.go

```diff
@@ -108,7 +108,7 @@ func runAddValidatorsSolo(_ context.Context, conf addValidatorsConfig) (err erro
 	}
 
 	// Perform a `gen_validators/v0.0.1` mutation using the newly created validators.
-	genVals, err := manifest.NewGenValidators(cluster.Hash, vals)
```

Comment:
Note this was a bug.

@xenowits can you please add a unit test that calls add_validators 2 times, so adds validators to the same manifest twice.